src/mpd.c

```diff
@@ -3589,6 +3593,24 @@ mpd_command_password(struct evbuffer *evbuf, int argc, char **argv, char **errms
   return ACK_ERROR_PASSWORD;
 }
 
+static int
+mpd_command_binarylimit(struct evbuffer *evbuf, int argc, char **argv, char **errmsg, struct mpd_client_ctx *ctx)
+{
+  unsigned int size;
+
+  if (safe_atou32(argv[1], &size) < 0)
+    {
+      DPRINTF(E_DBG, L_MPD,
+      	      "Argument %s to binarylimit is not a number\n",
+      	      argv[1]);
+      return ACK_ERROR_ARG;
+    }
+
```

Comment:
I see that for values < 64 bytes mpd errors with "Value too small". See ClientCommands.cxx. Maybe do the same?

Reply:
sensical, else stuff gets very inefficient – or even worse – we end up in an endless loop (size=0)